corehq/ex-submodules/casexml/apps/case/util.py

```diff
@@ -43,7 +48,34 @@ def post_case_blocks(case_blocks, form_extras=None, domain=None):
         **form_extras
     )
     response, xform, cases = sp.run()
-    return xform
+    return xform, cases
```

Comment:
I'm kind of surprised there weren't more references to this to change (though I assume you looked).

Not sure if it feels out of scope, but you could have `CaseFactory` do fewer DB calls by tweaking this function to leverage the new API and eliminate this call: https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/case/mock.py#L377